src/components/channel/channel.tsx

```diff
@@ -100,12 +100,16 @@ const useChannel = () => {
         }
         acc[date].push({
           body: message.Body,
-          sender: message.Sender,
+          // Update this section so that sender includes color and name
+          sender: {
+            name: message.Sender,
+            color: variant.Colors[message.Sender], // Only problem is that Colors isn't always defined for a variant!
```

Comment:
I know I added this change, but I think it breaks `Channel` unless the changes is followed through.

Reply:
I can't do anything with that - You'll need to fix this pull yourself then, :(